api/service/explorer/storage.go

```diff
@@ -74,85 +49,44 @@ func (storage *Storage) Init(ip, port string, remove bool) {
 	if remove {
 		var err = os.RemoveAll(dbFileName)
 		if err != nil {
-			utils.Logger().Error().Err(err).Msg("Failed to remove existing database files")
+			utils.Logger().Error().Err(err).Msg("Failed to re	move existing database files")
 		}
 	}
-	if storage.db, err = ethdb.NewLDBDatabase(dbFileName, 0, 0); err != nil {
+	options := &opt.Options{
+		OpenFilesCacheCapacity: 0,
```

Comment:
what's the meaning of those options here?  Can you add some more comments?

Reply:
Those are standard options for ethdb how it was created from leveldb.